packages/dcos-integration-test/extra/test_packaging.py

```diff
@@ -20,7 +23,7 @@ def test_pkgpanda_api(dcos_api_session):
         )
         pytest.skip(reason)
 
-    def get_and_validate_package_ids(path, node):
+    def get_and_validate_package_ids(path: str, node: Any) -> list:
```

Comment:
What type is `node`?

Reply:
As I can see from [documentation](https://dcos-test-utils.readthedocs.io/en/latest/dcos_test_utils.html#dcos_test_utils.dcos_cli.DcosCliConfiguration.get) method get using two parameters key and default. My assumption that node is default value in a case when path not found. I will use data type `str` here.